stripe: add page (#3962)
tldr-pages/tldr: pages/common/stripe.md

# stripe

> > Interact with your Stripe account.

Reviewer: with _a_ Stripe account

> More information: <https://github.com/stripe/stripe-cli>.

> - Log all activity on your account:

Reviewer: I believe this tails the logs right ? In that case "Log all activity" might be incorrect because it does not log, rather views the logs. Also, we should remove "your", as we avoid 2nd person in our pages.

Reviewer 2: Wow, I must have missed these

`stripe logs tail`

> - Listen for webhook events for `charge.succeeded` and forward them to localhost:3000/events:

Reviewer 2: Perhaps this will read better?
```suggestion
- Listen for events, filtering on events with the name `charge.succeeded` and forwarding them to localhost:3000/events:
```

> `stripe listen --events="charge.succeeded" --forward-to="localhost:3000/events"`

Reviewer 3: ```suggestion
`stripe listen --events="{{charge.succeeded}}" --forward-to="{{localhost:3000/events}}"`
```

We use bracket to indicate that these parts can be changed.

- Send a test webhook event:

> `stripe trigger charge.succeeded`

Reviewer 3: `charge.succeeded` can be replace by other option, right? So, wrap it with `{{}}`.

- Create a customer:

> `stripe customers create --email="test@example.com" --name="Jenny Rosen"`

Reviewer 3: ```suggestion
`stripe customers create --email="{{test@example.com}}" --name="{{Jenny Rosen}}"`
```

Same here.

> - Print to JSON and use jq to parse events:

Reviewer 2: Perhaps this will read better?
```suggestion
- Print to JSON and use `jq` to parse events:
```

> 

Reviewer: Let's remove the usage of `jq` here. It's not related to the command.
